app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,32 @@
+class SessionsController < ApplicationController
+  def new
+    if current_user
+      redirect_to home_path 
+    else
+      @user = User.new
+    end
+  end
+
+  def create
+    user = User.where(email: user_params[:email]).first
+    if user and user.authenticate(user_params[:password])
```

Comment:
You should rather use `&&` instead of `and` - as it has higher precedence. Read more [here](https://github.com/bbatsov/ruby-style-guide#no-and-or-or)